js/packages/common/src/contexts/meta/queryExtendedMetadata.ts

```diff
@@ -1,46 +0,0 @@
-import { MintInfo } from '@solana/spl-token';
```

Comment:
Since load accounts no longer fetches metadata I removed this from the initial store load function in meta context.